list-and-titles-of-products.md

```diff
@@ -0,0 +1,37 @@
+The purpose of this PR is to discuss and approve the list and correct titles of all products that we will indicate on the website in the near future.
+
+### Financial area
+
+Ubiquity Dollar 
+
+Ubiquity Card
+
+Ubiquity Payments
+
+### Management area
+
+DevPool
+
+DAO OS 
```

Comment:
Not sure if its knee-capping ourselves to call it "Ubiquity DAO Task OS" its a bit pigeonhole-y but it is more accurate I think. 

I would lean towards "Ubiquity OS" or "Ubiquitous(?)" because it can be the operating system that is all around us across all DAOs.